### ISSUE TITLE
[Bug]: DatePicker некорректные закругления при popupDirection="top"

### ISSUE THREAD

**Issue by danillas** [VKCOM/VKUI#6889]
### Описание

При popupDirection="top" закругления такие же как и при bottom

### Версия

6.0.2

### Шаги воспроизведения

Воспроизводится в доке https://vkcom.github.io/VKUI/#/DatePicker

Предусловия:
Должно быть достаточно места для открытия попапа сверху перед пикером:

`<View activePanel="datePicker">
  <Panel id="datePicker">
    <PanelHeader>DatePicker</PanelHeader>
    <Group>
    <Spacing size={130}/>
      <FormItem top="Дата рождения">
        <DatePicker
          min={{ day: 1, month: 1, year: 1901 }}
          max={{ day: 1, month: 1, year: 2006 }}
          onDateChange={(value) => {
            console.log(value);
          }}
          dayPlaceholder="ДД"
          monthPlaceholder="ММММ"
          yearPlaceholder="ГГГГ"
          popupDirection="top"
        />
      </FormItem>
    </Group>
  </Panel>
</View>`

### Ожидаемое поведение

Ожидается что будет наоборот - у верхних углов нет загруглений, у нижних есть

### Скриншоты

![image](https://github.com/VKCOM/VKUI/assets/26124760/ccb90256-17e0-4c5e-a251-0bd1f45271a8)

**Comment by levtsypanov** [comment]
У меня не работает `popupDirection="top"`. Остаётся по дефолту `"bottom"`
![2024-05-06_15-12-20](https://github.com/VKCOM/VKUI/assets/51300421/5727396d-dc0f-4121-98a0-73f3dcafd0de)
![2024-05-06_15-12-02](https://github.com/VKCOM/VKUI/assets/51300421/73914a3d-59a0-4f97-a4e9-6a01fd4b1d3b)

**Comment by danillas** [comment]
[levtsypanov](https://github.com/levtsypanov) См. предусловия и приложенный код. Там есть дополнительный спейсинг в 130px

**Comment by mendrew** [comment]
@levtsypanov 

> У меня не работает popupDirection="top". Остаётся по дефолту "bottom"

Да, он будет работать как `top`, если сверху достаточно места для дропдауна.
